lib/class/xml_data.class.php

```diff
@@ -221,18 +221,19 @@ private static function playlist_song_tracks_string($song, $playlist_data)
     public static function keyed_array($array,$callback='')
     {
         $string = '';
-
+        return $array;
```

Comment:
Yes, anyway we will still be able to remove them when the PR is ready and that you will squash your commits. Not really important at the moment. Plus, use whatever makes your debugging easier.